src/systems/actions.cairo

```diff
@@ -48,7 +48,7 @@ pub mod actions {
 
             // 2. Set the player's remaining moves to 100.
             let moves = Moves {
-                player, remaining: 100, last_direction: Direction::None(()), can_move: true
+                player, remaining: 100, last_direction: core::Option::None, can_move: true
```

Comment:
```suggestion
                player, remaining: 100, last_direction: Option::None, can_move: true
```
would be great to avoid having to import core this way is possible.